dev/release/verify-release-candidate.sh

```diff
@@ -382,6 +382,11 @@ install_go() {
     return 0
   fi
 
+  if which go > /dev/null 2>&1; then
+    show_info "Found $(go version) at $(which go)"
```

Comment:
Could you use `command -v` instead of `which` because `command -v` is implemented in all POSIX compatible shells but `which` is an external command?

```suggestion
  if command -v go > /dev/null; then
    show_info "Found $(go version) at $(command -v go)"
```

(I know that we already use `which` in this script. :-)

Reply:
Right, I tend to forget about other shells :)